projects/components/src/table/data/table-cdk-row-util.ts

```diff
@@ -151,6 +151,20 @@ export namespace TableCdkRowUtil {
     return [row, ...children];
   };
 
+  export const selectAllRows = (rows: StatefulTableRow[]): StatefulTableRow[] =>
```

Comment:
It looks like we only ever use unselect. If selectAll isn't used, can we remove it? Or did I miss where it was used

Reply:
right. removing it